projects/step-core/src/lib/modules/tree/components/tree/tree.component.scss

```diff
@@ -1,5 +1,20 @@
 cdk-tree-node {
   display: flex;
+  position: relative;
+
+  .node-line {
+    position: absolute;
+    top: 0;
+    // level * paddingIdent + level * paddingMultiplier + (24px - 2px) / 2
+    left: calc(var(--level) * 24px + var(--level) * 8px + 11px);
```

Comment:
Here css variable is suitable, as it take value from angular's binding. Looks fine.